spec/ics-003-connection-semantics/README.md

```diff
@@ -306,6 +317,7 @@ function connOpenInit(
 ```typescript
 function connOpenTry(
   desiredIdentifier: Identifier,
+  provedIdentifier: Identifier,
```

Comment:
couldn't this just be a boolean indicating if the sending chain decided to choose the identifier or not?

Reply:
sure, though we'll still construct a "provedIdentifier" since we need to check it